common/usr/share/MailScanner/perl/MailScanner/Message.pm

```diff
@@ -3449,6 +3449,11 @@ sub UnpackRar {
     # not a RAR file, fail to extract
     return 1 if $ArchiveInfo =~ /^\s*No files to extract\s*$/mi;
 
+    if ($ArchiveInfo =~ /^\s*Checksum error in the encrypted file .*? Corrupt file or wrong password\./) {
+      MailScanner::Log::WarnLog("Unrar: Unable to list files without password");
+      return "password";
+    }
+
```

Comment:
Why are you adding this whole stanza instead of just augmenting the
```
if ($ArchiveInfo =~ /encrypted headers|password is incorrect/i) {
```
test below.

Reply:
Oh, my bad, I am sorry. This is my mistake, just forgot remove the rule for the host.
Please, reject and close the pull request.